scripts/configure-venue-access.ts

```diff
@@ -0,0 +1,119 @@
+import fs from "fs";
+import admin from "firebase-admin";
+import serviceAccount from "./prodAccountKey.json";
+import "firebase/firestore";
+import { Venue } from "../src/types/Venue";
+import {
+  VenueAccessCodes,
+  VenueAccessEmails,
+  VenueAccessPassword,
+  VenueAccessType,
+} from "../src/types/VenueAcccess";
+
+const mergeStringArrays = (
+  arr1: string[] | undefined,
+  arr2: string[] | undefined
+): string[] => {
+  const result = (arr1 ?? []).map((val) => val.trim());
+  (arr2 ?? []).forEach((val) => {
+    const valTrimmed = val.trim();
+    if (!result.includes(valTrimmed)) {
+      result.push(valTrimmed);
+    }
+  });
+  return result;
+};
+
+function usage() {
+  console.log(`
+${process.argv[1]}: Configure venue access. Supports configuring a secret password, list of emails which can access the venue, or ticket codes.
+
+Usage: npx ts-node ${process.argv[1]} PROJECT_ID VENUE_ID [password|emails|codes] [password | emails file path | codes file path]
+
+Example: npx ts-node ${process.argv[1]} co-reality-map password abc123
+Example: npx ts-node ${process.argv[1]} co-reality-map emails emails-one-per-line.txt
+Example: npx ts-node ${process.argv[1]} co-reality-map codes ticket-codes-one-per-line.txt
+`);
+  process.exit(1);
+}
+
+const argv = process.argv.slice(2);
+if (argv.length < 4) {
+  usage();
+}
+
+const projectId = argv[0];
```

Comment:
`const [projectId, venueId, method, accessDetail] = argv;`

Reply:
https://github.com/sparkletown/sparkle/pull/1016#discussion_r559297221

Reply:
It was using the method mentioned above, but you commented that it's not consistent with the rest of the scripts and I changed it back.

Reply:
That may have been a miscommunication/misunderstanding then, I think I may have been saying that some other aspect wasn't consistent.

Have a look at https://github.com/sparkletown/sparkle/pull/1065, that's how we want them to be, eg.

```typescript
const [projectId, collection, credentialPath] = process.argv.slice(2);

// Note: no need to check credentialPath here as initFirebaseAdmin defaults it when undefined
if (!projectId || !collection) {
  usage();
}

```